app/views/purchases/new.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= simple_form_for [ @book, @purchase ] do |f| %>
```

Comment:
I'm not sure we need @purchase in Simple form,  because it is not in the URL of the Routes new
/books/:book_id/purchases/new(.:format)